### ISSUE TITLE
2018 Content Spec: AMD 1: Clarify language regarding compatibility brand in section 3.4

### ISSUE THREAD

**Issue by johnsim** [cta-wave/content-specification-task-force#7]
(Section 3.4) Clarity – replace phrase “Players implementing … compatibility brand” with “Players using … compatibility brand.” Chris Poole note: “It wasn’t clear what implementing a brand would mean (does it mean recognising it, or does it mean supporting the means to decode it, for example). Suggest “using” or re-writing this more clearly. Defer for now because there was a group decision on the use of the term ‘implementing’ which had previously been ‘support’. Requires further discussion.

**Comment by johnsim** [comment]
Made edit changes recommended by @poolec

**Comment by KilroyHughes** [comment]
I took the action to add text to clarify that Baseline profile is very restricted, and why. And, that new splice constraints will likely be specified and why … so readers better understand what to expect. Combined text below. Added text in italics.

This document specifies the WAVE Baseline Splice Constraint Profile, but other Splice Constraint Profiles could be specified in the future. _WAVE Baseline Splice Constraints require very restrictive encoding parameters and Fragment sequencing to enable smooth playback on the majority of adaptive streaming devices deployed in the market. Less restrictive splice constraint profiles will likely be specified in the future that will provide smooth sequencing on more advanced devices._
